pyat/test/test_lattice_object.py

```diff
@@ -0,0 +1,184 @@
+import sys
+import numpy
+import pytest
+from at import elements
+from at.load import load_mat
+from at.lattice import Lattice, AtWarning, AtError
+
+
+def test_lattice_creation():
```

Comment:
Some of these tests are really two or three tests bundled into one.

It's probably better to split them. You end up with very long test names, but people do like that style:
http://blog.socosomi.com/dont-be-afraid-of-long-test-names/

Reply:
Ok, I'll split some of them down.